### PR TITLE
Auto-update coverage highlighting

### PR DESCRIPTION
**Pull request by emilienlemaire**
This PR should enable coverage to work with our `superbol-studio` extension

### CODE REVIEW

File: src/coverage.ts [hunk 1]
```diff
@@ -14,8 +14,8 @@ import {
 import * as os from "os";
 import * as nativePath from "path";
 import * as ChildProcess from "child_process";
-import {SourceMap} from "./parser.c";
-import {GcovData, loadGcovData} from "./gcov";
+import { SourceMap } from "./parser.c";
```
Comment (nberth) [inline src/coverage.ts]: Although I am all for this kind of readability changes, we sometimes want to be able to grab fixes from the original/forked repository.
To do that more easily, unfortunately we need to avoid any unnecessary change, even white-space ones.

File: src/coverage.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ export class CoverageStatus implements Disposable {
         outline: 'none',
         backgroundColor: 'rgba(20, 250, 20, 0.2)'
     });
-    readonly COMMAND = 'gdb.coverage-toggle';
+    private readonly COMMAND = 'gdb.coverage-toggle';
```
Comment (nberth) [inline src/coverage.ts]: Same as above: is this change of visibility required?

File: src/gdb.ts [hunk 3]
```diff
@@ -100,22 +99,22 @@ export class GDBDebugSession extends DebugSession {
...
-                this.miDebugger.emit("ui-break-done");
-            }, 50);
-            this.sendResponse(response);
-            this.miDebugger.start().then(() => {
-                this.started = true;
-                if (this.crashed)
-                    this.handlePause(undefined);
-            }, (err: Error) => {
-                this.sendErrorResponse(response, 100, `Failed to start MI Debugger: ${err.toString()}`);
+        /*onfulfilled:*/() => {
```
Comment (nberth) [inline src/gdb.ts]: Please revert the changes in indentation.

**Review by nberth — COMMENTED** [review]
Would pushing those changes upstream make sense? In this case the "style" fixes could stay. Otherwise, I'm afraid changes need to be kept to a minimum to facilitate future merges.

**Comment by emilienlemaire** [comment]
It turn out my linter autorunned on save, and I did not check all the changes before committing, I'll revert to only the logical changes in a few

**Review by nberth — APPROVED** [review]
Thanks! Please squash before merge.
Oh, and edit the title for a proper description (like "Auto-update displayed coverage")?
